lib/linguist/languages.yml

```
@@ -1778,6 +1778,16 @@ Elvish:
   tm_scope: source.elvish
   color: "#55BB55"
   language_id: 570996448
+Elvish transcript:
+  type: programming
+  ace_mode: text
+  extensions:
+  - ".elvts"
+  interpreters:
+  - elvish-transcript
+  tm_scope: source.elvish-transcript
+  color: "#99FF99"
+  language_id: 452025714
```

This can be simplified:

```suggestion
Elvish Transcript:
  type: programming
  group: Elvish
  ace_mode: text
  tm_scope: source.elvish-transcript
  color: "#55BB55"
  language_id: 452025714
```

Notable changes:

- we don't need an extension as it'll only ever be used in markdown files
- we group it under Elvish as it is essentially the results of processing that language, like `pycon` for Python
- we don't need an interpreter as this is only used for shebang matching which will never happen
- the colour, whilst never likely to be seen, should probably be the same as Elvish